app/assets/javascripts/discourse/tests/integration/components/simple-list-test.js

```diff
@@ -48,6 +48,24 @@ module("Integration | Component | simple-list", function (hooks) {
     );
   });
 
+  test("adding a value when list is predefined", async function (assert) {
+    this.set("values", "vinkas\nosama");
+    this.set("choices", ["vinkas", "osama", "kris"]);
+
+    await render(
+      hbs`<SimpleList @values={{this.values}} @allowAny={{false}} @choices={{this.choices}}/>`
+    );
+
+    await click(".add-value-input summary");
+    await click(".select-kit-row");
```

Comment:
I wonder if we just add one more assertion here to see that there is only one value in the list of choices, since the other two have already been set?